src/sonic-yang-models/yang-models/sonic-buffer-profile.yang

```diff
@@ -0,0 +1,82 @@
+module sonic-buffer-profile {
+    namespace "http://github.com/Azure/sonic-buffer-profile";
+    prefix bpf;
+
+    import sonic-buffer-pool {
+        prefix bpl;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC BUFFER PROFILE";
+
+    revision 2021-07-01 {
+        description
+            "Initial revision.";
+    }
+
+
+    container sonic-buffer-profile {
+
+        container BUFFER_PROFILE {
+
+            list BUFFER_PROFILE_LIST {
+                key "name";
+
+                leaf name {
+                    type string;
+                    description "Buffer Profile name";
+                }
+
+                leaf static_th {
+                    type uint64;
+                    description "The maximum size of the buffer pool the port can occupy";
+                }
+
+                leaf dynamic_th {
+                    type int32 {
+                        range "-7..3";
```

Comment:
I think the range should be the union of the legal range of all vendors 
`-8..7` should be better.
BTW the bounders are inclusive, right?